custom_code/hooks.py

```diff
@@ -139,7 +139,7 @@ def target_post_save(target, created):
             if np.isfinite(dist):
                 target.distance = dist
             disterr = hostdict[0].get('DistErr', np.nan)
-            if np.isfinite(disterr):
+            if np.isfinite(disterr[0]):
                 target.distance_err = disterr
```

Comment:
Last question: does this work on both cases (if `disterr` is a float or a list)? Remember that this code block only runs if `target.distance` has not previously been set, but you can manually delete the distance from a target for testing.

Reply:
Yes you were right, I fixed this by removing the indexing and adding a `np.all` call around it!

Reply:
There was still an issue with storing an array in the `target.distance_err` attribute, but I just put a `np.mean` around it. (This isn't used for anything, so it doesn't really matter.)